### ISSUE TITLE
possibly removing `qtconsole` from jupyter metapackage

### ISSUE THREAD

**Issue by ivanov** [jupyter/qtconsole#598]
Hey `qtconsole` users, I wanted to start a conversation about the possibility of removing `qtconsole` from the base `jupyter` metapackage.  

I've opened [a thread with a poll on the Jupyter Discourse](https://discourse.jupyter.org/t/poll-and-feedback-on-possibly-removing-qtconsole-from-jupyter-metapackage/22882) and you can also chime in on [the Jupyter metapackage issue](https://github.com/jupyter/jupyter/issues/718)

This would affect you if you don't currently install `qtconsole` directly, and rely on it being installed when you install `jupyter`, so I'd like to get a sense of how many people would be affected by such a change, and what your thoughts around that are.

**Comment by ccordoba12** [comment]
As the maintainer of this package, I agree with this decision. Not only this would remove a heavy dependency on the `jupyter` metapackage, but I think regular users mostly want to use notebooks when they install Jupyter, not other clients that use the Jupyter execution architecture.

**Comment by fperez** [comment]
I also agree - I think at this point the web console in JupyterLab serves this need for most purposes, and the value of simplifying/making lighter our base installation seems like the right tradeoff today.

I strongly support this idea.

**Comment by ivanov** [comment]
Circling back to close this. `jupyter` 1.1.1, released in August 2024, removed `qtconsole` from its dependencies, though  `qtconsole` continues to be available for users needing it, they now need to install it explicitly. Thanks everyone!
